storage-service/src/index.ts

```diff
@@ -165,12 +165,7 @@ app.post(
       });
       res.send({ docId, secret: result }).end();
     })().catch((err) => {
-      if (err.code === "NoSuchBucket") {
-        req.log.error(
-          { err },
-          "NoSuchBucket at /upload. Please restart storage-service to create a new bucket at minio/container in Azure blob storage",
-        );
-      }
+      req.log.error({ err }, "Error while uploading document");
       res.status(500).send(err).end();
```

Comment:
@SamuelPull I would consider sending whole err object to the user as unsafe. But this is probably not part of this task.

Reply:
I agree 100%.  Client here is usually not a user, but an API server, and I didn't write the line you're referring to. I just added logging in a terminal of storage-service.  Moreover, at this point, I think it's more important for users to have some information they can report to us